nevergrad/benchmark/experiments.py

```diff
@@ -418,7 +418,7 @@ def powersystems(seed: Optional[int] = None) -> Iterator[Experiment]:
     algos = ["NaiveTBPSA", "SQP", "Powell", "LargeScrHammersleySearch", "ScrHammersleySearch", "PSO", "OnePlusOne",
              "CMA", "TwoPointsDE", "QrDE", "LhsDE", "Zero", "StupidRandom", "RandomSearch", "HaltonSearch",
              "RandomScaleRandomSearch", "MiniDE", "SplitOptimizer5", "SplitOptimizer9", "SplitOptimizer", "SplitOptimizer3", "SplitOptimizer13"]
-    for budget in [25, 50, 100, 200, 400, 800, 1600, 3200, 6400, 12800]:
+    for budget in [1600, 3200, 6400, 12800]:
```

Comment:
Less budget values, because the first values were pointless.
It's not a problem of computational cost, it's a problem of having significant results.